core/server/services/bulk-email/index.js

```diff
@@ -46,24 +47,45 @@ module.exports = {
      * @param {Email} message - The message to send
      * @param {[EmailAddress]} recipients - the recipients to send the email to
      * @param {[object]} recipientData - list of data keyed by email to inject into the email
-     * @returns {Promise<boolean>} A promise representing the success of the email sending
+     * @returns {Array{Promise<boolean>}} An array of promises representing the success of the batch email sending
      */
     async send(message, recipients, recipientData) {
+        let BATCH_SIZE = 1000;
+
         if (!mailgunInstance) {
             return;
         }
         let fromAddress = message.from;
-        if (/@localhost$/.test(message.from)) {
+        if (/@localhost$/.test(message.from) || /@ghost.local$/.test(message.from)) {
```

Comment:
Added `ghost.local` as it's something I commonly use but don't know if anybody else is having a similar setup? :)

Reply:
Did mailgun error with `ghost.local`?

Reply:
It does fail with `Error: 'from' parameter is not a valid address` message.